docs/source/api/standalone.mdx

```diff
@@ -168,6 +168,7 @@ import { ApolloServerPluginDrainHttpServer } from '@apollo/server/plugin/drainHt
 import express from 'express';
 import http from 'http';
 import cors from 'cors';
+import bodyParser from 'body-parser';
```

Comment:
`body-parser` actually comes with express now, so the `bodyParser` usage should just become `express.json`. We did this in the express middleware docs but must've missed this one.

Reply:
Good point, did not know that. I updated the PR